packages/gatsby-source-shopify/README.md

```diff
@@ -127,6 +131,18 @@ To create a production build, use gatsby build
 
 Now follow the second link to explore your Shopify data!
 
+<div id="priority-builds"></div>
+
+### Priority builds
+
+Because of the limiations of the Shopify Bulk API we have included logic in this plugin to determine which builds are high priority for a given Shopify site. This allows us to pause deploy preview builds while production builds are running while using the same Shopify App. The following logic determines whether a build is priority or not:
```

Comment:
Shouldn't this be "cancel"?

```suggestion
Because of the limiations of the Shopify Bulk API we have included logic in this plugin to determine which builds are high priority for a given Shopify site. This allows us to cancel preview builds while production builds are running while using the same Shopify App. The following logic determines whether a build is priority or not:
```